Dockerfile

```diff
@@ -0,0 +1,112 @@
+# Stage 1:
```

Comment:
I don't think this should live in the root directory ?

Reply:
Where were you thinking? Currently, `docker build .` works as expected from the root directory.

Reply:
Can you rename it to say... .ci.Dockerfile? Then you can just pass in the -f argument with docker build I think.

Reply:
To add to that, dockerhub let's you configure the location of the Dockerfile for automatic builds as well.